.github/workflows/build-release-binary.yml

```diff
@@ -0,0 +1,129 @@
+name: "Build release binary"
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build_binaries:
+    name: Build release binary
+    strategy:
+      matrix:
+        include:
+          - bin: taker
+            target: x86_64-unknown-linux-gnu
+            os: ubuntu-latest
+            archive_ext: tar
```

Comment:
out of curiosity: why `.tar` and not more conventional `.tar.gz`?

Reply:
We are not compressing things so it is just an archive!